elements/urth-viz-scatter/urth-viz-scatter.html

```diff
@@ -31,6 +31,17 @@
     Polymer({
         is: 'urth-viz-scatter',
 
+        properties: {
+            /**
+             * Allows user to select multiple points
+             */
+            multiSelect: {
```

Comment:
No need for property here once it is moved to `elements/urth-viz-chart/urth-viz-chart-common-behavior.html`.